src/main/java/Duke.java

```diff
@@ -3,53 +3,60 @@
 
 public class Duke {
     public static void main(String[] args) {
-        //Implementing Increment Level-0: Initial skeleton of Duke
+        //Starting the bot
         System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
-        Task List[] = new Task[100];
+        Task List[] = new Task[100];  //Creating a list of tasks
         int listSize = 0;
-        while(true){
 
-            String line;
+        while(true) {
+            System.out.println("--------------------------------------------------------------"); //Formatting
+            String line; //Input from user
             Scanner in = new Scanner(System.in);
             line = in.nextLine();
 
-            if(line.split(" ")[0].equals("bye")){
+            if(line.split(" ")[0].equals("bye")) {
```

Comment:
Leave a space in between the if and condition
if (condition) {
    statements;
}

Reply:
You've included most of the main logic inside this `if else` block, which could make this function extremely long. I would recommend extracting out each command into smaller functions, which would also make the overall code more readable